native/src/routes/feedback/components/FeedbackModal.js

```diff
@@ -60,33 +84,34 @@ class FeedbackModal extends React.Component<PropsType> {
           <Description theme={theme}>{t('feedbackType')}</Description>
           <Picker
             selectedValue={feedbackOptions.indexOf(feedbackItem)}
-            onValueChange={this.props.onFeedbackOptionChanged}
+            onValueChange={props.onFeedbackOptionChanged}
             mode='dropdown'>
             {feedbackOptions.map((item, index) => (
               <Picker.Item label={item.label} value={index} key={index} />
             ))}
           </Picker>
+
           <Description theme={theme}>
             {' '}
             {isPositiveFeedback ? t('positiveComment') : t('negativeComment')}
             {!isPositiveFeedback && <RequiredText>*</RequiredText>}
           </Description>
-          <Input
-            theme={theme}
-            onChangeText={this.props.onCommentChanged}
-            autoFocus
-            value={comment}
-            multiline
-            placeholderTextColor={theme.colors.textSecondaryColor}
-            placeholder={t('yourFeedback')}
-          />
+          <Input theme={theme} onChangeText={props.onCommentChanged} value={comment} multiline />
```

Comment:
Why did you remove the `autofocus` and `placeholder` props?

Reply:
I oriented myself on the design in zeplin:
https://zpl.io/VQZPQek

Autofocus and placeholders are removed.

Reply:
I'd keep the autofocus at least. I also liked the placeholder text.

Reply:
If we leave autofocus, then the keyboard is open when opening this site. When we remove it, I think it is a lot clearer, because when on opening the page you get an overview, which fields are there and see the send button. 
![feedback](https://user-images.githubusercontent.com/18463686/111600578-d8ca8e00-87d1-11eb-9b36-8c52323393e4.png)